collectors/python.d.plugin/anomalies/README.md

```diff
@@ -1,15 +1,16 @@
 <!--
 title: "Anomalies"
+description: "Use ML-driven anomaly detection to narrow your focus to only affected metrics and services/processes on your node to shorten root cause analysis."
 custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/anomalies/README.md
 -->
 
-# Anomalies: ML-driven anomaly detection for the Netdata Agent
```

Comment:
I think the more descriptive this can be the better since anomalies seem to be a wide-ranging term